less/mixins.less

```diff
@@ -1 +1,22 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.btn-frame(@bg-color: @header-font-color, @width: 100px){
+    background-color: @bg-color;
+    color: white;
+    padding: 1rem;
+    width: @width;
+    text-align: center;
+    border-radius: 10px;
+    font-size: 1.8rem;
+    font-weight: bolder;
+
+    &:hover{
+        background-color: lightblue;
```

Comment:
Tip: add a color parameter, and use variables here to switch the background and font colors.  This would help the mixin be more reusable if you had different color buttons on the page